lib/jsdom/browser/Window.js

```diff
@@ -1,13 +1,13 @@
 "use strict";
 
 var CSSStyleDeclaration = require("cssstyle").CSSStyleDeclaration;
-var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+var file = require("./file");
```

Comment:
Can you follow the style used in `living`, putting these files there, instead of putting these as siblings in `jsdom/browser`? Example:

https://github.com/tmpvar/jsdom/blob/master/lib/jsdom/living/url.js plus https://github.com/tmpvar/jsdom/blob/master/lib/jsdom/living/index.js#L30